yjit_iface.c

```diff
@@ -375,7 +411,7 @@ rb_yjit_cme_invalidate(VALUE cme)
         st_table *block_set = (st_table *)blocks;
 
         // Invalidate each block
-        st_foreach(block_set, block_set_invalidate_i, 0);
+        st_foreach(block_set, invalidate_block_key_i, (st_data_t)INVAL_METHOD_LOOKUP);
```

Comment:
The answer is probably no, but couldn't you just ask for the length of the block set and increment the counter directly?

Reply:
Ah yeah that's better.